packages/frontend/src/components/Editor/index.tsx

```diff
@@ -257,19 +230,19 @@ export default function Editor(props: EditorProps): React.ReactElement {
     <Flex w="full" justifyContent="center">
       <Flex flexDir="column" alignItems="center" py={3} w="53.25rem">
         <StepExecutionsToIncludeProvider value={stepExecutionsToInclude}>
-          {stepsBeforeGroup.map((step, index, steps) => (
+          {stepsBeforeGroup.map((step, index) => (
             <Fragment key={`${step.id}-${index}`}>
               <FlowStep
                 step={step}
+                allEditorSteps={steps}
```

Comment:
can actually pass in 
```
isLastStep={index === steps.length - 1}
```
like in L250

Reply:
passing in the entire steps array seems a bit heavy handed since it _may_ cause excessive re-renders